.github/workflows/Python.yml

```diff
@@ -82,7 +82,7 @@ jobs:
       run: |
         export DISTUTILS_C_COMPILER_LAUNCHER=ccache
         # TODO: Use ccache inside container, see https://github.com/pypa/cibuildwheel/issues/1030
-        cibuildwheel --output-dir wheelhouse --config-file cibw.toml duckdb_tarball
+        cibuildwheel --output-dir wheelhouse --config-file pyproject.toml duckdb_tarball
```

Comment:
Bit of a nitpick, but this is the default value for config-file

Reply:
Ah alright, if there are other issues or the PR requires touching in another way I'll address it, if not I'll leave it as is 👍